vagrant-kubernetes-istio/setup_vm.sh

```diff
@@ -44,4 +44,3 @@ vagrant reload
 vagrant ssh -c "kubectl get pods -n kube-system"
 vagrant ssh -c "mkdir ~/.kube/"
```

Comment:
Can we remove this line as it error's out saying this directory already exists..
By the time we reach here this directory should already exist..

Reply:
On my testing, I got the error that the ~/.kube/ dir does not exist.